sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/interface.scala

```diff
@@ -180,7 +180,8 @@ case class CatalogTable(
       Seq(s"Table: ${identifier.quotedString}",
         if (owner.nonEmpty) s"Owner: $owner" else "",
         s"Created: ${new Date(createTime).toString}",
-        s"Last Access: ${new Date(lastAccessTime).toString}",
+        "Last Access: " +
+          (if (lastAccessTime == -1) "UNKNOWN" else new Date(lastAccessTime).toString),
```

Comment:
I don't feel strongly about it, but seems like elsewhere an empty string is used for no values. This seems slightly preferable.